DESCRIPTION

```diff
@@ -5,12 +5,16 @@ Authors@R: c(
     person(given = "Joshua W.", family = "Lambert", email = "joshua.lambert@lshtm.ac.uk", role = c("aut", "cre", "cph"),
            comment = c(ORCID = "0000-0001-5218-3046")),
     person(given = "Carmen", family = "Tamayo", email = "carmen.tamayo-cuartero@lshtm.ac.uk", role = c("aut"),
-           comment = c(ORCID = "https://orcid.org/0000-0003-4184-2864"))
-  )
+           comment = c(ORCID = "https://orcid.org/0000-0003-4184-2864")),
+    person(given = "Hugo", family = "Gruson", email = "hugo.gruson@data.org", role = c("ctb"),
```

Comment:
```suggestion
    person(given = "Hugo", family = "Gruson", email = "hugo@data.org", role = c("ctb"),
```

Reply:
Thanks. I pulled the incorrect email address from another Epiverse package. Will go through the other Epiverse packages I maintain to ensure the email is correct. 

I just noticed that your email in the {packagetemplate} is different, do you have a preference for which to use in packages?